templates/beego/files/conf/conf.go

```diff
@@ -0,0 +1,16 @@
+package conf
+
+import (
+  psh "github.com/platformsh/config-reader-go/v2"
+)
+
+var Config = createConfigObj()
```

Comment:
Assuming people are going to use this file for other things, this should be named less generically.  PshConfig or similar.  (And then update the factory function and references to it.)

Reply:
Also, I have no idea which is more Go-ish; making this call directly here, or defining the variable and then populating it from `init()`.  Absent knowledge of that I can go either way, but calling it out in case you've seen some hint.